### PR TITLE
View users list

### PR DESCRIPTION
**Pull request by Phvanhoang**
image

## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/35524)

## WHAT
- View list of trainees and supervisors

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by Phvanhoang** [comment]
Ready

File: app/controllers/supervisor/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Supervisor::UsersController < SupervisorController
+  authorize_resource
+
+  def trainees_index
+    @q = User.ransack(params[:q])
+    @trainees = @q.result.page(params[:page]).per_page(Settings.page)
+                  .trainee.sort_by_created_at_desc
+                  .includes(:user_exams)
```
Comment (vanvtt-0952) [inline app/controllers/supervisor/users_controller.rb]: sort đồ xong xuôi mới phân trang nhé chú

File: app/controllers/supervisor/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  end
+
+  def supervisors_index
+    @q = User.ransack(params[:q])
+    @supervisors = @q.result.page(params[:page]).per_page(Settings.page)
+                     .supervisor.sort_by_created_at_desc
```
Comment (vanvtt-0952) [inline app/controllers/supervisor/users_controller.rb]: ` .supervisor.sort_by_created_at_desc` tương tự comment trên

**Comment by Phvanhoang** [comment]
Ready

**Pull request merged**